pkgs/development/libraries/ffmpegthumbnailer/default.nix

```diff
@@ -1,15 +1,15 @@
-{ fetchFromGitHub, lib, stdenv, ffmpeg, cmake, libpng, pkg-config, libjpeg
+{ fetchFromGitHub, fetchpatch, lib, stdenv, ffmpeg, cmake, libpng, pkg-config, libjpeg
```

Comment:
`fetchpatch` leftover from testing?

Other than that, lgtm.

Reply:
Oh, good catch! Yeah, a remnant of local change. Removed.